File: JAVA-CarRacingProject/ProiectCurse/src/main/java/repository/interfaces/IParticipantRepository.java
package repository.interfaces;

import domain.Participant;
import repository.ICrudRepository;

public interface IParticipantRepository extends ICrudRepository<Integer, Participant> {
    /**
     * @param echipa - echipa dupa care se filtreaza participantii
     * @return toti participantii filtrati
     */
    Iterable<Participant> findParticipantibyEchipa(String echipa);

}

File: C#-CarRacingProject/Repository/Interfaces/IParticipantRepository.cs
using ProiectCSharpVersion.Domain;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ProiectCSharpVersion.Repository.Interfaces
{
    interface IParticipantRepository:ICrudRepository<int,Participant>
    {
        //filtrare dupa echipa
        IEnumerable<Participant> GetParticipantiByEchipa(string echipa);
    }
}
